src/samplers/support/transform.jl

```diff
@@ -191,8 +198,8 @@ Distributions.logpdf{T}(d::SimplexDistribution, x::Vector{T}, transform::Bool) =
   if transform
     K = length(x)
     z = Vector{T}(K-1)
-    for k in 1:K-1
-      z[k] = x[k] / (one(T) - sum(x[1:k-1]))
+    @simd for k in 1:K-1
```

Comment:
I'm not sure whether we should include `@simd` in Turing, since `@simd` is experimental according to Julia documentation https://docs.julialang.org/en/release-0.4/manual/performance-tips/